# Ilia Popov

[![Telegram](https://img.shields.io/badge/Telegram--brightgreen?style=social&logo=Telegram)](https://t.me/ilya_pop0v)
[![iljapopov17@gmail.com](https://img.shields.io/badge/iljapopov17@gmail.com--brightgreen?style=social&logo=gmail)](mailto:iljapopov17@gmail.com)

[Bioinformatics Institute](https://bioinf.me/en), Bioinformatics for Biologists </br>
[Rostov State Medical University](https://rostgmu.net), Medical Doctor </br>

## Research experience

<details><summary>
<b>Bioinformatics Institute project</b>: The shadow of HIV: searching for indirect signs of HIV infection in cell-free DNA samples
</summary><br> 
 
Data of WGS-based non-invasive prenatal testing (NIPT) or cell-free DNA testing contains exogenous DNA (bacterial and viral). This information is too fragmentary to conduct full microbiome studies, but still interesting for expanding NIPT functionality.<br>
Being a retrovirus, HIV can not be directly detected in cell-free DNA data.<br>
Steps of the study:<br>
1) Extraction of unmapped reads
2) **__Assigning taxonomic labels__** - current step
3) Creating residual virus and microbiome profiles of two datasets
4) Analysis of the HIV-positive sequencing data
5) Finding the differences in exogenous DNA composition between HIV- and HIV+ NIPT samples<br>

The results will be published in the very begining of the May, 2024 in [The-shadow-of-HIV repository](https://github.com/iliapopov17/The-shadow-of-HIV).<br>
  * **Skills:** Bash, bwa, bowtie2, Snakemake, kraken2.<br>
</details>

<details><summary>
<b>Research Internship</b>: Pediatric Nutritional Medicine and Intestinal Microbiology
</summary><br>

Performed genome-based safety assessment of the probiotic strain _Lpb. plantarum_ IS-10506.<br>
Determined the multivariate association between clinical metadata and microbial meta-omics characteristics in a clinical study comparing gut microbiota profiles in stunted and normal children aged 36-45 months.<br>
Details available in [PNMIM repository](https://github.com/iliapopov17/PNMIM)
  * **Skills:** BAGEL4, CRISPRCasFinder, R, dplyr, ggplot2, tidyverse, tidyr, MaAsLin2.<br>
</details>

## Study projects

<details><summary>
<b>Beginner-level tools</b>
</summary><br> 
 
1. **SequenceForge-Lite**<br>
[Repository](https://github.com/iliapopov17/SequenceForge-Lite)<br>
It is a lightweight tool designed to work with biological sequence data, providing various functionalities for filtering FASTQ files and manipulating FASTA files

2. **MyAwesomeEDA**<br>
[Repository](https://github.com/iliapopov17/MyAwesomeEDA)<br>
Python module that provides a set of tools for exploring and analyzing your dataset
</details>

<details><summary>
<b>Bioinformatics Institute: mini-projects</b>
</summary><br> 

 This [repository](https://github.com/iliapopov17/BI-Workshop-miniProjects) holds mini projects completed as part of the workshops undertaken during the training at the Bioinformatics Institute. Currently, it describes work on bacterial genome annotation, assembly, protein function prediction, genotyping and RNA sequencing data analysis to detect differential expression
</details>

<details><summary>
<b>Bioinformatics Institute: Phylogenetics</b>
</summary><br>
 
This [repository](https://github.com/iliapopov17/BI-Phylogenetics) collects a complete pipeline of simple research in phylogenetics, from working with NCBI (and other databases) to building trees, evaluating them, and getting some worthwhile results.
</details>

<details><summary>
<b>Human Microbiome Research</b>
</summary><br>
 
This [repository](https://github.com/iliapopov17/Human-Microbiome-Research-User-Manual) contains a report on a project that was carried out as part of the training course «Human Microbiome Research - User Manual» by Blastim
</details>

**Many more to be disclosured soon!**

## Skills: 

<p align="left">
   <a 
    href="https://www.python.org" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/Python-FFD43B?style=for-the-badge&logo=python&logoColor=blue"
    /> 
 </a> 
     <a 
    href="https://pandas.pydata.org" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/Pandas-2C2D72?style=for-the-badge&logo=pandas&logoColor=white"
    /> 
 </a> 
   <a 
    href="https://numpy.org" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/Numpy-777BB4?style=for-the-badge&logo=numpy&logoColor=white"
    /> 
 </a> 
   <a 
    href="https://code.visualstudio.com/" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/VSCode-007ACC.svg?&style=for-the-badge&logo=VisualStudioCode&logoColor=white"
    />    
 </a> 
 <a 
    href="https://posit.co/download/rstudio-desktop/" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/R-276DC3?style=for-the-badge&logo=r&logoColor=white"
    /> 
 </a>
    <a 
    href="https://posit.co/download/rstudio-desktop/" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/RStudio-75AADB?style=for-the-badge&logo=RStudio&logoColor=white"
    /> 
 </a> 
  <a 
    href="https://jupyter.org" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/Jupyter-F37626?style=for-the-badge&logo=jupyter&color=525252"
    /> 
 </a> 
    <a 
    href="https://anaconda.org" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/Anaconda-44A833?style=for-the-badge&logo=anaconda&color=525252"
    /> 
 </a>
    <a 
    href="https://ubuntu.com" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/Ubuntu-E95420?style=for-the-badge&logo=ubuntu&logoColor=white"
    /> 
 </a> 
       <a 
    href="https://www.overleaf.com" 
    target="_blank" rel="noreferrer"> 
    <img
        src="https://img.shields.io/badge/Overleaf-47A141?style=for-the-badge&logo=Overleaf&logoColor=white"
    /> 
 </a> 
</p>
